Core/tests/Unit/Logger/AppEngineFlexHandlerTest.php

```diff
@@ -34,7 +35,12 @@ public function setUp()
     {
         $dir = sys_get_temp_dir();
         $this->stream = tmpfile();
-        $handler = new AppEngineFlexHandler(Logger::DEBUG, true, 0640, false, $this->stream);
+        if (Logger::API === 1) {
```

Comment:
```suggestion
        if (!defined('Monolog\Logger::API') || Logger::API === 1) {
```

If we check whether the constant is defined, it'll allow us to leave a wider range open in the 1.x require field.

Reply:
Oh, that's good!

Reply:
Done!